demosplan/DemosPlanCoreBundle/Controller/User/DemosPlanOrganisationAPIController.php

```diff
@@ -360,6 +365,17 @@ public function updateOrgaAction(
                 // explicitly set that show list may be updated
                 $userHandler->setCanUpdateShowList(true);
             }
+
+            if ($permissions->hasPermission('feature_orga_new_procedure_edit') && is_array($orgaDataArray['attributes'])
```

Comment:
Why did you put it here? Feels wrong to me. This should live in Permissions imho

Reply:
done here 36ce1db346c5c2cc0057a1d44f6a60611e3b7cff